### ISSUE TITLE
Audio Glitching on Realtek Speakers

### ISSUE THREAD

**Issue by gidgeesheep** [Facepunch/sbox-issues#5400]
### Describe the bug

Sbox's audio is wrong (see video below) when played out of the main speakers of my Laptop. Using a different audio device on the same system is not causing problems.

### To Reproduce

1. Have my laptop?? ROG Strix G513QY_G513QY and play S&Box using the main audio Realtek Speakers
2. Play any sound, especially sudden, or loud sounds.
3. Notice it is wrong.
4. Plug in any other audio device to fix it.

### Expected behavior

The audio should play correctly without glitching noises.

### Media/Files

Video with sound from Realtek Driver

https://github.com/Facepunch/sbox-issues/assets/55063683/570e7fb9-978c-426e-beb2-d6b39f0d49af

Switching to pair of headphones or any other audio device

https://github.com/Facepunch/sbox-issues/assets/55063683/204756f1-3a26-4564-9d50-c07d73d57844

The device causing trouble
![Screenshot 2024-04-22 124701](https://github.com/Facepunch/sbox-issues/assets/55063683/be9149bb-4844-40b7-ad72-b4099ac59a8a)
<img width="302" alt="Screenshot 2024-04-22 125605" src="https://github.com/Facepunch/sbox-issues/assets/55063683/9d9ce155-6cce-405d-a29b-3226ef5e2fba">

### Additional context

OS Name	Microsoft Windows 11 Home
Version	10.0.22631 Build 22631
Other OS Description 	Not Available
OS Manufacturer	Microsoft Corporation
System Name	SUPER-DUPER-LAP
System Manufacturer	ASUSTeK COMPUTER INC.
System Model	ROG Strix G513QY_G513QY
System Type	x64-based PC
System SKU	
Processor	AMD Ryzen 9 5980HX with Radeon Graphics, 3301 Mhz, 8 Core(s), 16 Logical Processor(s)
BIOS Version/Date	American Megatrends International, LLC. G513QY.331, 2/24/2023
SMBIOS Version	3.3
Embedded Controller Version	0.82
BIOS Mode	UEFI
BaseBoard Manufacturer	ASUSTeK COMPUTER INC.
BaseBoard Product	G513QY
BaseBoard Version	1.0
Platform Role	Mobile
Secure Boot State	On
PCR7 Configuration	Elevation Required to View
Windows Directory	C:\WINDOWS
System Directory	C:\WINDOWS\system32
Boot Device	\Device\HarddiskVolume3
Locale	United States
Hardware Abstraction Layer	Version = "10.0.22621.2506"
User Name	SUPER-DUPER-LAP\Christian
Time Zone	Eastern Daylight Time
Installed Physical Memory (RAM)	64.0 GB
Total Physical Memory	63.4 GB
Available Physical Memory	48.2 GB
Total Virtual Memory	67.4 GB
Available Virtual Memory	46.4 GB
Page File Space	4.00 GB
Page File	C:\pagefile.sys
Kernel DMA Protection	On
Virtualization-based security	Not enabled
Windows Defender Application Control policy	Enforced
Windows Defender Application Control user mode policy	Off
Device Encryption Support	Elevation Required to View
Hyper-V - VM Monitor Mode Extensions	Yes
Hyper-V - Second Level Address Translation Extensions	Yes
Hyper-V - Virtualization Enabled in Firmware	No
Hyper-V - Data Execution Protection	Yes

**Comment by shurahgolden** [comment]
i could totally be wrong about this, but my microphone can sometimes output a similar glitching/static sound when the bitrate and or sample rate aren't the same.
can you try making sure that the settings (bitrate & sample rate) are set to either: 44100Hz 16Bits, or 48000Hz 16Bits?
also make sure that you have nothing enabled in the Room Correction tab and the Sound Effects tab. Also, if this still didn't fix anything than try turning your speaker configuration to Stereo mode, even if your speakers _may_ be truly compatible with other modes, it's worth just checking.
![ApplicationFrameHost_2yzLGho289](https://github.com/Facepunch/sbox-issues/assets/87254883/36cf9921-3b58-43e4-9c93-599b6573fd77)
lemme me know if any of this fixed anything, and if so what did!

**Comment by gidgeesheep** [comment]
No, no changes in the speaker frequency made a change. It's just 48000Hz 16Bits permanently. But the problem remains.

**Comment by James76931** [comment]
I've been getting this bug for AGES... not with Realtek speakers but with my MOTU M2 audio interface. Has only ever been an issue in s&box.

**Comment by JochemvanHout** [comment]
> I've been getting this bug for AGES... not with Realtek speakers but with my MOTU M2 audio interface. Has only ever been an issue in s&box.

Same issue, also on an MOTU m2 usb audio interface.

**Comment by RenTec** [comment]
Same issue also on Realtek but with a 3.5 jack headset on the front of my desktop

**Comment by DolphinOfficial** [comment]
Can confirm, I use Realtek and I have the same crackling/popping. Really annoying. This wasn’t happening a year ago, by the way.
